server/liveblog/embed/embed_assets/themes/default-theme/package.json

```diff
@@ -1,16 +1,17 @@
 {
     "name": "default-theme",
+    "label": "Default Theme",
     "version": "1.0",
     "angularModule": "default-theme",
     "styles": [
+        "styles/reset.css",
         "styles/embed.css",
-        "//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css",
-        "//cdnjs.cloudflare.com/ajax/libs/animate.css/3.3.0/animate.min.css",
-        "//fonts.googleapis.com/css?family=Poppins:400,500"
+        "http://fonts.googleapis.com/css?family=Roboto:400,300"
     ],
     "scripts": [
-        "//code.angularjs.org/1.3.14/angular-sanitize.js",
         "//code.angularjs.org/1.3.14/angular-animate.js",
+        "vendors/ng-infinite-scroll.min.js",
+        "vendors/fecha.js",
```

Comment:
Ah yes true... I'm used to the lighweight fecha, I forgot we are already
importing moment.js.
But after this sprint, I will remove all theses implicit imports. So I
think we can keep this one

On Fri, Jun 19, 2015 at 11:34 AM nistormihai <notifications@github.com>
wrote:

> In server/liveblog/embed/embed_assets/themes/default-theme/package.json
> <https://github.com/superdesk/liveblog/pull/123#discussion_r32814478>:
>
> >          "//code.angularjs.org/1.3.14/angular-animate.js",
> > +        "vendors/ng-infinite-scroll.min.js",
> > +        "vendors/fecha.js",
>
> why? isn't moment already here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/liveblog/pull/123/files#r32814478>.
>